docs/receiving/using-app-portal/adding-endpoints.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Adding Endpoints
+---
+
+In order to start listening to messages sent through Svix, you will need to configure your **endpoints**.
+
+Adding an endpoint is as simple as providing a URL that you control and a list of **event types** that you want to
+listen to.
+
+![add endpoint](/img/app-portal/add-endpoint.png)
+
+:::info Faster integration with Svix Play
```

Comment:
I wouldn't call it integration. More like testing?

Reply:
Users integrate with us

Reply:
Reworded for clarity:

`Integrate faster using Svix Play`

Reply:
sounds good. removed the word integrate.